docs/cloud/satellites/best-practices.md

```diff
@@ -0,0 +1,90 @@
+# Satellites Best Practices
+
+Earthly Satellites are remote runner instances provided by the Earthly team, offering the ability to perform builds in the cloud while retaining cache between runs. This best practices guide aims to help users optimize their workflow and leverage the full potential of Earthly Satellites.
+
+## Size Satellites for Builds
+
+Properly sizing Earthly Satellites is crucial to achieving optimal performance and resource utilization. Consider the following when determining which Satellite size is appropriate for your use case.
+
+### 1. Resource Requirements of a Single Build
+
+Understand the resource demands of your build, including memory usage, CPU requirements, and required cache size on disk. 
+
+For example, builds utilizing computationally intensive compilation steps, or large integration tests may require additional CPU and memory. Builds that download many gigabytes of dependencies may require a large amount of cache on disk.
```

Comment:
can we point out `earthly --exec-stats +target` in some way here (or elsewhere in this doc?)

Reply:
Good idea. Added this as a tip below. 

P.S. I was going to link it to a doc page on the `--exec-stats` page but noticed we didn't have one.